log/logrus_logger.go

```diff
@@ -0,0 +1,34 @@
+package log
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/sirupsen/logrus"
+)
+
+type logrusLogger struct {
+	*logrus.Logger
+}
+
+// NewLogrusLogger returns a logger that logs the keyvals to Writer
+// with a time stamp at the logrus.InfoLevel
+func NewLogrusLogger(w io.Writer) Logger {
+	newLogger := logrus.New()
```

Comment:
I believe the main use case for this package is to adapt a `*logrus.Logger` used as the main logger in an application so it can be passed to code that expects a Go kit `log.Logger`. For that use case this constructor would be more flexible if it were declared as `NewLogrusLogger(logger *logrus.Logger) log.Logger`.

This form allows the main application to configure logrus however it wants and then simply adapt it to the Go kit `log.Logger` interface.